### DISCUSSION TITLE
Multiple Nodes in same line

### DISCUSSION THREAD

**Discussion by Jei-sKappa** [AppFlowy-IO/appflowy-editor#107] [Q&A]
Hi, I'm new to AppFlowy and wondered if it could render several nodes in the same line.
I tried to create my own NodeWidget and make it render itself and its children in a Row by accessing the children property in the node but by doing so the children are rendered by Appflowy in another line. Can you tell me how I can do it?
Example
json = 
{
	"type":"text",
	"attributes":{
		"subtype":"myCustomNode"
	},
	"delta":[
		{
			"insert":"ParentText"
		}
	],
	"children":[
		{
			"type":"text",
			"delta":[
				{
					"insert":"Child One"
				}
			]
		},
		{
			"type":"text",
			"delta":[
				{
					"insert":"Child Two"
				}
			]
		}
	]
}.

Expected output:
Parent <someinlinewidget> Child One <someinlinewidget> Child Two

Current output
Parent <someinlinewidget> Child One <someinlinewidget> Child Two
    Child One
    Child Two

Thanks!!

**Comment by LucasXu0** [comment]
@Jei-sKappa In this case, you should customize the builder for your own type instead of passing the node's children to render service.
